src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,187 @@
+                        if THE_PROBABLE_LOGIC_NER_CHECK[0] in _values:
+                            df.loc[index,'is_copyright'] = "t"
+        else:
+            df.loc[index,'is_copyright'] = "f"
+        
+        if clutter_flag == 1:
+            if df.loc[index,'is_copyright'] == "t":
+                clutterRemoval(df, index, listB)
+    return
+
+def clutterRemoval(df, index, ner_list):
+    string1 = "all rights reserved"
+    string2 = "distributed under the mit software license"
+    string3 = df.loc[index,'content']
+
+    if string1 in string3:
+        clutter_removed = string3[:string3.index(string1)]
+        df.loc[index,'edited_text'] = clutter_removed 
+
+    elif string2 in string3:
+        clutter_removed = string3[:string3.index(string1)]
+        df.loc[index,'edited_text'] = clutter_removed
+    
+    elif 'ORG' in ner_list['Values'] and 'PERSON' in ner_list['Values']:
+        org_name = ner_list['Entity'][ner_list['Values'].index('ORG')]
+        person_name = ner_list['Entity'][ner_list['Values'].index('PERSON')]
+        try:
+            if string3.index(org_name) > string3.index(person_name):
+                clutter_removed = string3[:string3.index(org_name)] + org_name
+                df.loc[index,'edited_text'] = clutter_removed
+            else:    
+                clutter_removed = string3[:string3.index(person_name)] + person_name
+                df.loc[index,'edited_text'] = clutter_removed
+        except:
+            df.loc[index,'edited_text'] = string3
+
+    elif 'ORG' in ner_list['Values']:
+        org_name = ner_list['Entity'][ner_list['Values'].index('ORG')]
+        try:
+            string3 = re.sub(r'''[^\w\s()]+''', '', string3)
+            clutter_removed = string3[:string3.index(org_name)] + org_name
+            df.loc[index,'edited_text'] = clutter_removed
+        except:
+            df.loc[index,'edited_text'] = string3
+
+    elif 'PERSON' in ner_list['Values']:
+        person_name = ner_list['Entity'][ner_list['Values'].index('PERSON')]
+        try:
+            clutter_removed = string3[:string3.index(person_name)] + person_name
+            df.loc[index,'edited_text'] = clutter_removed
+        except:
+            df.loc[index,'edited_text'] = string3
+
+    
+    else:
+        clutter_regex = r"(Copyright|copyright\s*(©)?([\w \-\,\[\]]+)?(\.com)?\.?|Copyright\s*(©)?|©([\w \-\,\.\[\]]+)?(Copyright)?)|(Copyright\s*(\(c\))?|\(c\)\s*(Copyright)?(?:[\w \,\-\.\"\[\]]{2,53}\s*)?\.?|\(c\)\s*(Copyright)?)|Copyright|([\w,]|(\s*\d+(\s(?:\,|-)\s*\d+)?\s*))(\s*\d+(\s*(?:\,|-)\s*\d+)?\,?\s*)\s*[a-zA-Z\&\| ,\s0-9]{3,50}(\.com)?\.?|(\.|\,)?\s*(\@[^>]*?\.com)|Inc+(\.)?| Company+|Corporation+|& Co+|GmbH+|All rights reserved(?:\.)?|Ltd|\<|[\w.]+@[a-zA-Z0-9-_.]+|\>|([A-Za-z0-9]+\.com)|rights\s*reserved|(?:(?:https?|ftp|file):\/\/|www\.|ftp\.)(?:\([-\w+&@#/%=~|$?!:,.]*\)|[-\w+&@#/%=~|$?!:,.])*(?:\([\w+&@#/%=~|$?!:,.]*\)|[\w+&@#/%=~|$])|\<|\>|\(|\)"
```

Comment:
Please make this regex also a constant at the top of the script. It will make it easier to locate the regex if someone needs modifications to be done.